app/assets/stylesheets/ltags/StackexchangeTag.scss

```
@@ -141,22 +145,11 @@
     z-index: var(--z-elevate);
     text-align: center;
     position: relative;
-    box-shadow: 0px 0px 60px 42px #fff;
-    box-shadow: 0px 0px 60px 42px var(--card-bg);
+    box-shadow: 0 0 60px 42px var(--card-bg);
   }
 
   .ltag__stackexchange--btn {
-    color: #0366d6;
-    background-color: #f1f8ff;
-    border-radius: 3px;
-    line-height: 20px;
-    padding: 0.25em 1.2em;
-    opacity: 0.9;
-    font-weight: bold;
-    border: 1px solid #0366d6;
-    font-size: 0.75em;
-    &:hover {
-      opacity: 1;
-    }
+    @extend .crayons-btn !optional;
+    @extend .crayons-btn--secondary !optional;
```

```suggestion
    @extend .crayons-btn--secondary !optional;
    
    // A deprecated version of this liquid tag's HTML included a link _inside_ a button :-/
    // This `revert` is needed to make sure the button click still opens the content
    a {
      pointer-events: revert;
    }
```

Everything is looking great, but I noticed that the main call to action button doesn't work for an old post, with the new CSS. It seems like styles from `crayons-btn` are preventing pointer events on the link that's nested inside the button.

Having a link inside a button is terrible HTML, which is why I thought it best we add a comment here as well 🙃  The happy news is your HTML changes will mean any future versions of this liquid tag won't have this issue, and I'm sure one day we can eventually remove this hack 😄